Update README.md (#44)
DataDog/datadog-static-analyzer-github-action: README.md

| `architecture` | The CPU architecture to use for the analyzer. Supported values are `x86_64` and `aarch64`.                               | No      | `x86_64`        |
| `diff_aware` | Enable [diff-aware scanning mode][5].                                                                                      | No      | `true`          |

## Customizing rules

> By default, [Datadog Static Analyzer][8] detects the languages of your codebase and pull the default rulesets to analyze

Reviewer: ```suggestion
By default, [Datadog Static Analyzer][8] detects the languages of your codebase and uses the default rulesets to analyze
```

your codebase.

> If you want to specify the rulesets to use (and customize them), add a `static-analysis.datadog.yml` file to your 
> repository's root directory to specify which rulesets to use.

Reviewer: ```suggestion
To specify and customize the rulesets, add a `static-analysis.datadog.yml` file to your repository's root directory to define which rulesets to use.
```

```yaml
rulesets:
  - <ruleset-name>
  - <ruleset-name>
```

> The list of all rulesets is available in [Datadog documentation][6].

Reviewer: ```suggestion
Refer to the [Datadog documentation][6] for a complete list of rulesets.
```

### Example for Python

> You can see an example for Python-based repositories:

Reviewer: ```suggestion
Here is an example for Python-based repositories:
```
